MeesGame/Gameplay/GameStates/GameOverState.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Xna.Framework;
+using Microsoft.Xna.Framework.Graphics;
+using Microsoft.Xna.Framework.Input;
+
+namespace MeesGame.Gameplay.GameStates
+{
+    class GameOverState : IGameLoopObject
+    {
+        public GameOverState()
+        {
+
+        }
+
+        //Upon pressing space, return to the menu/editor
+        //TODO: Make the user go back to the editor if he came from there
+        public void HandleInput(InputHelper inputHelper)
+        {
+            if (!inputHelper.KeyPressed(Keys.Space))
+                return;
```

Comment:
Rework the if statement so that an else is not needed. Simply remove '!' and 'return; else'